src/s3-resource-selector/s3-modal/styles.scss

```diff
@@ -5,6 +5,10 @@
 @use '../../internal/styles/tokens.scss' as awsui;
 @use '../../internal/styles' as styles;
 
+.modal-root {
+  /* used in test-utils */
+}
```

Comment:
You added this test-utility in a file which may generate unstable selectors when we refactor the code.

Did you want to put it under a new file `src/s3-resource-selector/test-classes/styles.scss` which is going to be a stable location for all utils moving forward?

Reply:
First time seeing the pattern. Do we want to move all the test util classes here as part of this PR or follow up PR?

Reply:
Well,

<img width="623" alt="image" src="https://github.com/user-attachments/assets/bf4d0927-5189-4a91-a201-857bd30f61a9">

I think our current strategy is to make sure all new classes get added according to this convention. Updating past selectors is something that @orangevolon is looking at.